crates/prelude/src/lib.rs

```diff
@@ -51,6 +51,8 @@ pub mod button;
 mod callback;
 #[cfg(feature = "internal-api-crypto")]
 pub mod crypto;
+// TODO: Add feature
+pub mod ctap;
```

Comment:
This should be under `usb` like `serial` (except it doesn't use the generic `Serial` API).

Reply:
Done.